### ISSUE TITLE
Java inter-op with varargs of array

### ISSUE THREAD

**Issue by Biruntha** [ballerina-platform/ballerina-lang#20061]
**Description:**
Java inter-op is not accepting the varargs of array

B7a code
```
function nativeBatchUpdate(Param?[]... parameters) returns BatchUpdateResult = @java:Method {
class: "org.ballerinax.jdbc.actions.BatchUpdate"
} external;
```
Java method signature
```
public static MapValue<String, Object> nativeBatchUpdate(ArrayValue... parameters)
```

**Steps to reproduce:**

**Affected Versions:**

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by SupunS** [comment]
@Biruntha  Whats the error you are getting?

Anyway can you test with the master branch now?

**Comment by Biruntha** [comment]
I haven't get any errors. I'm getting this Assertion failure.  

ballerina-jdbc-test-suite > ballerina-jdbc-test > org.ballerinax.jdbc.actions.BatchUpdateTest > testBatchUpdate FAILED
    java.lang.AssertionError: expected [1] but found [-3]
        at org.testng.Assert.fail(Assert.java:96)
        at org.testng.Assert.failNotEquals(Assert.java:776)
        at org.testng.Assert.assertEqualsImpl(Assert.java:137)
        at org.testng.Assert.assertEquals(Assert.java:118)
        at org.testng.Assert.assertEquals(Assert.java:547)
        at org.testng.Assert.assertEquals(Assert.java:557)
        at org.ballerinax.jdbc.actions.BatchUpdateTest.testBatchUpdate(BatchUpdateTest.java:64)

**Comment by SupunS** [comment]
I believe thats a problem with the test case.

Closing the issue since this original issue is fixed with #20015
